Keywords/pages/Page_Licensure_Unit_Appeals.groovy

```diff
@@ -20,38 +22,120 @@ import com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords as WebUI
+	private TestObject h1AppealSubmitted		= findTestObject('Page_Licensure_Unit_Appeals/h1_Your appeal has been submitted')
 
-	private TestObject inputAppealCode						= findTestObject('Page_Licensure Unit Appeals/input_Appeal Code')
-	private TestObject buttonNext							= findTestObject('Page_Licensure Unit Appeals/button_Next')
-	private TestObject inputEmail							= findTestObject('Page_Licensure Unit Appeals/input_Applicant Email')
-
+	/*************
+	 * Inputs
+	 *************/	
 	@Keyword
 	def enterAppealCode (String appealCode) {
 		WebUI.waitForElementVisible(inputAppealCode, 0)
 		WebUI.sendKeys(inputAppealCode, appealCode)
 	}
 
 	@Keyword
-	def clickOnNextButton () {
-		WebUI.waitForElementVisible(buttonNext, 0)
-		WebUI.click(buttonNext)
+	def enterUpdatedPhone (String phone) {
+		WebUI.waitForElementVisible(inputPhone, 0)
+		WebUI.setText(inputPhone, phone)
+	}
+	@Keyword
+	def enterUpdatedPhonExt (String ext) {
+		WebUI.waitForElementVisible(inputPhoneExt, 0)
+		WebUI.setText(inputPhoneExt, ext)
+	}
+
+	@Keyword
+	def enterUpdatedAddressL1 (String address) {
+		WebUI.waitForElementVisible(inputAddressL1, 0)
+		WebUI.setText(inputAddressL1, address)
+	}
+	@Keyword
+	def enterUpdatedAddressL2 (String address) {
+		WebUI.waitForElementVisible(inputAddressL2, 0)
+		WebUI.setText(inputAddressL2, address)
+	}
+
+	@Keyword
+	def enterUpdatedAddressL3 (String address) {
+		WebUI.waitForElementVisible(inputAddressL3, 0)
+		WebUI.setText(inputAddressL3, address)
+	}
+
+	@Keyword
+	def enterUpdatedZip (String zip) {
+		WebUI.waitForElementVisible(inputZip, 0)
+		WebUI.setText(inputZip, zip)
+	}
+
+	@Keyword
+	def enterAppealReason (String text) {
+		WebUI.waitForElementVisible(textareaAppealReason, 0)
+		WebUI.sendKeys(textareaAppealReason, text)
+	}
+
+	/*************
+	 * Buttons
+	 *************/
+	@Keyword
+	def clickOnNextBtn () {
+		WebUI.waitForElementVisible(btnNext, 10)
+		WebUI.click(btnNext)
+	}
+
+	@Keyword
+	def clicOnCloseModalBtn () {
+		WebUI.waitForElementVisible(btnModalClose, 10)
+		WebUI.click(btnModalClose)
+	}
+
+	@Keyword
+	def clicOnModalSubmitBtn () {
+		WebUI.waitForElementVisible(btnModalSubmit, 10)
+		WebUI.click(btnModalSubmit)
+	}
+	
+	@Keyword
+	def clicOnSubmitBtn () {
+		WebUI.waitForElementVisible(btnSubmit, 10)
+		WebUI.click(btnSubmit)
 	}
 
+	/****************
+	 * Validations
+	 ****************/
 	@Keyword
-	def verifyEmail (String email) {
-		WebUI.waitForElementVisible(inputEmail, 0)
-		WebUI.verifyElementAttributeValue(inputEmail, 'value', email, 0)
+	def verifyAppealSubmitted () {
+		WebUI.waitForElementVisible(h1AppealSubmitted, 10)
+		if (!WebUI.verifyElementPresent(h1AppealSubmitted, 10)) {
+			KeywordUtil.markFailed("\nThe test case can not be completed. Error alert won't show due INVALID denial code entered")
+		}
 	}
 
 	@Keyword
 	def verifyErrorAlert (String alertObj, String errorText) {
+		WebUI.waitForElementVisible(alertObj, 10)
```

Comment:
@fedecl-ot , as we saw, you are waiting for a String. alertObj is not an Object Repository

Reply:
Done